src/universal-dapp.js

```diff
@@ -598,10 +618,19 @@ UniversalDApp.prototype.getCallButton = function (args) {
     })
   }
 
+  var title
+  if (isConstructor) {
+    title = 'Create'
```

Comment:
Does it make sense to name it (Create Contract), (Deploy Contract) or (Create Instance) ..?
so it does not overlap with a potential `Create` function